internal/util/indexcgowrapper/build_index_info.go

```diff
@@ -65,6 +66,7 @@ func NewBuildIndexInfo(config *indexpb.StorageConfig) (*BuildIndexInfo, error) {
 		access_key_value: cAccessValue,
 		root_path:        cRootPath,
 		storage_type:     cStorageType,
+		cloud_provider:   cCloudProvider,
```

Comment:
Compatibility needs to be taken into consideration here. If the cloudProvider field in the indexpb.StorageConfig passed is empty, it is necessary to fallback to the original logic.

Reply:
In the case of it being empty, shall we consider setting the default value as "minio"?